apps/site/next.config.js

```diff
@@ -44,6 +44,17 @@ const nextConfig = {
 				destination: DOCUSIGN_FORM_URL,
 				permanent: true,
 			},
+			{
+				source: "/incident",
+				destination:
+					"https://docs.google.com/forms/d/e/1FAIpQLSehTGeZxoYy5-zQ-dBeEXjvAzy2kc25X8-74LBWbYbW-xMJNQ/viewform",
```

Comment:
Please use https://forms.gle/A6BdsSzYSiyeTP8Y6 for the incident response form instead. I was able to get this by going to the edit view of the form, clicking "Send", then clicking the link tab and "Shorten URL".

Reply:
I do not have access to the edit view, so maybe that's why